### DISCUSSION TITLE
Idp Ldap Edit-mode

### DISCUSSION THREAD

**Discussion by T2brozz** [zitadel/zitadel#6236] [Q&A]
Hey I found your project and it looks awesome.
But I have a question. We have a read-only LDAP. Currently, we use Keycloak and there we set it to [Edit-Mode: unsynced](https://wjw465150.gitbooks.io/keycloak-documentation/content/server_admin/topics/user-federation/ldap.html). 

Is this possible with zitadel?

**Comment by fforootd** [comment]
I think it should work since we currently only do ldap binding request for authentication.
The data we copy for the user is stored as zitadel user afterwards so only the authentication happens directly to the ldap after the first login.

Give it a spin and let us know! 😁

**Comment by T2brozz** [comment]
Ok thanks, I'm already trying to set it up
